doc/api/net.md

```diff
@@ -889,6 +905,7 @@ For TCP connections, available `options` are:
 * `port` {number} Required. Port the socket should connect to.
 * `host` {string} Host the socket should connect to. **Default:** `'localhost'`.
 * `localAddress` {string} Local address the socket should connect from.
+  This is ignored if `autoDetectFamily` is set to `true`.
```

Comment:
Does enabling it by default break applications that rely on `localAddress` etc.? The documentation says that the option will be ignored when `autoDetectFamily` is enabled.

Reply:
I feel like it should be the opposite: ignore `autoDetectFamily` if `localAddress` is specified. Unless there's another solution that doesn't make these options mutually exclusive.

Reply:
Yes, I don't like ignoring pre-existing options either.

Reply:
Agreed, we should ignore `autoDetectFamily` if `localAddres` is set.

Reply:
Changing it.